#!/usr/bin/env python
PACKAGE = "mobility_sensing"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pc_RobustThreshold", int_t, 0, "Points needed for a wall",    300, 10,   5000)
gen.add("pc_Resolution", int_t, 0, "Fraction of points to use from the pointcloud (1/pc_Resolution points are used)",    5, 1,   20)
gen.add("pc_vertThreshold", double_t, 0, "How vertical wall needs to be (0 = must be perfectly up and down, 1 = can be perfectly horizontal)", .2, .05, 1)
gen.add("pc_planeDistThreshold", double_t, 0, "Max distance points in plane can be from eachother (don't quite understand this yet)", .03, .01, .1)

exit(gen.generate(PACKAGE, "mobility_sensing", "ClosestWall"))
